lib/middleware/websocket/interactors/build_payload.rb

```diff
@@ -0,0 +1,19 @@
+module Websocket
+  module Interactor
+    class BuildPayload
+      attr_reader :payload
+
+      def initialize(clients: clients)
+        @clients = clients
+        @payload = perform_build
+      end
+
+      def perform_build
```

Comment:
A more common pattern for service objects is to call the method that "does the things" `#call`.

Reply:
@swcraig Yeah I was wondering about that, I was just trying to remove an extra line of code. In this case instead of:
```
      def generate_payload
        BuildPayload.new(clients: @clients).payload
      end
```
We would have:
```
      def generate_payload
        build_payload = BuildPayload.new(clients: @clients)
        build_payload.call
      end
```

Reply:
@swcraig This has been updated, good suggestions.